pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -22,12 +22,16 @@ import (
 	"io"
 	"net"
 	"net/http"
+	ossys "os"
```

Comment:
Since 'openstack' is no longer aliased to 'os' in this file, can we remove the 'ossys' alias?